core-java-modules/core-java-14/pom.xml

```diff
@@ -28,25 +34,26 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <release>${maven.compiler.release}</release>
-                    <compilerArgs>--enable-preview</compilerArgs>
-                    <source>14</source>
-                    <target>14</target>
+                    <compilerArgs>
+                        <arg>--add-modules</arg>
+                        <arg>jdk.incubator.foreign</arg>
+                    </compilerArgs>
```

Comment:
@vunamtien 
source/target aren't needed, since we build this module with default-jdk9 profiles which contain source/target 17 already. You can verify with the mvn command plus "-X"